docs/extensions/availableExtensions.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 10
+---
+
+# Available Extensions
+
+## Core Extensions
+
+Core extensions are extensions build and maintained by Scaffold-ETH 2 team. This extensions can be directly accessed via just extension name without mentioning the github user name and branch name:
+
+```bash
+npx create-eth@latest -e <extension-name>
+```
+
+E.g.: `npx create-eth@latest -e subgraph`
+
+Here are the core extensions:
+
+- [subgraph](https://github.com/scaffold-eth/create-eth-extensions/tree/subgraph): This Scaffold-ETH 2 extension helps you build and test subgraphs locally for your contracts. It also enables interaction with the front-end and facilitates easy deployment to Subgraph Studio.
+- [eip-712](https://github.com/scaffold-eth/create-eth-extensions/tree/eip-712): An implementation of EIP-712, allowing you to send, sign, and verify typed messages in a user-friendly manner.
+- [ponder](https://github.com/scaffold-eth/create-eth-extensions/tree/ponder): This Scaffold-ETH 2 extension comes pre-configured with [ponder.sh](https://ponder.sh), providing an example to help you get started quickly.
+- [onchainkit](https://github.com/scaffold-eth/create-eth-extensions/tree/onchainkit): This Scaffold-ETH 2 extension comes pre-configured with [onchainkit](https://onchainkit.xyz/), providing an example to help you get started quickly.
+- [erc-20](https://github.com/scaffold-eth/create-eth-extensions/tree/erc-20): This extension introduces an ERC-20 token contract and demonstrates how to interact with it, including getting a holder balance and transferring tokens.
```

Comment:
In the subgraph, ponder and onchainkit maybe I'd give some extra context so you can guess better if any of these match your needs (newb developers maybe haven't heard of some of them)

Since we would be adding more text, maybe we can get rid of "Scaffold-ETH 2 extension.." to make them shorter, and add it to the previous sentence: `Here are the core extensions of Scaffold-ETH 2:`

Examples:

- [subgraph](https://github.com/scaffold-eth/create-eth-extensions/tree/subgraph): Uses subgraph from The Graph to index and query blockchain data. Helps you build and test subgraphs locally for your contracts. It also enables interaction with the front-end and facilitates easy deployment to Subgraph Studio.
- [ponder](https://github.com/scaffold-eth/create-eth-extensions/tree/ponder): Pre-configured with [ponder.sh](https://ponder.sh),  an open-source framework for blockchain application backends. With Ponder, you can rapidly build & deploy an API that serves custom data from smart contracts on any EVM blockchain.
- [onchainkit](https://github.com/scaffold-eth/create-eth-extensions/tree/onchainkit): Pre-configured with [onchainkit](https://onchainkit.xyz/), providing an example to help you get started quickly with the ready-to-use React components and Typescript utilities built by Coinbase team.